plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -207,6 +218,26 @@
 
     private static final int MAX_SUPPORTED_TEMPORAL_PRECISION = 7;
 
+    private static final PredicatePushdownController SQLSERVER_STRING_COLLATION_AWARE_PUSHDOWN = (session, domain) -> {
```

Comment:
The name is a bit misleading - it's just case-sensitivitiy aware, not collation (ordering) aware. So a simpler name like `SQLSERVER_CHARACTER_PUSHDOWN` might be better.

Reply:
Fixed